packages/uui-tabs/lib/uui-tab-group.element.ts

```diff
@@ -24,58 +30,292 @@ export class UUITabGroupElement extends LitElement {
       ::slotted(*:not(:last-of-type)) {
         border-right: 1px solid var(--uui-tab-divider, none);
       }
+
+      .hidden-tab {
+        width: 100%;
+      }
+
+      #hidden-tabs-container {
+        width: fit-content;
+        display: flex;
+        flex-direction: column;
+        background: var(--uui-color-surface);
+        border-radius: var(--uui-border-radius);
+        box-shadow: var(--uui-shadow-depth-3);
+        --uui-tab-text-align: left;
+      }
+      :host([dropdown-direction='horizontal']) #hidden-tabs-container {
+        flex-direction: row;
+      }
+
+      #more-button {
+        margin-left: auto;
+        position: relative;
+      }
+      #more-button::before {
+        content: '';
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        background-color: var(--uui-color-current);
+        height: 0px;
+        border-radius: 3px 3px 0 0;
+        opacity: 0;
+        transition: opacity ease-in 120ms, height ease-in 120ms;
+      }
+      #more-button.active-inside::before {
+        opacity: 1;
+        height: 4px;
+        transition: opacity 120ms, height ease-out 120ms;
+      }
     `,
   ];
 
+  @query('#more-button')
+  private _moreButtonElement!: UUIButtonElement;
+
   @queryAssignedElements({
     flatten: true,
     selector: 'uui-tab, [uui-tab], [role=tab]',
   })
   private _slottedNodes?: HTMLElement[];
-  private _tabElements: HTMLElement[] = [];
 
-  private _setTabArray() {
-    this._tabElements = this._slottedNodes ? this._slottedNodes : [];
+  /**
+   * Toggles the priority navigation mode, which will collapse tabs into a dropdown when there is not enough space.
+   * @type {boolean}
+   * @attr
+   * @default false
+   */
+  @property({ type: Boolean, reflect: true, attribute: 'priority-navigation' })
+  priorityNavigation = false;
```

Comment:
Totally loose idea, but how would it be if it was always true?

Reply:
Just from the thinking of mobile friendly and extendable, then its not fore sure that the implementation of the tabs group nesecary knows the items of it, or can be clever about the space..